src/components/App.svelte

```diff
@@ -1,7 +1,31 @@
 <script>
-  export let name = 'Svelte'
+  import Header from './Header';
+  import Slider from './Slider';
+  import TestSlide from './TestSlide';
+  import Footer from './Footer';
 </script>
 
-<h1 class="text-red-500 text-5xl text-center font-hairline py-8">
-  Hello, it's me, {name}!
-</h1>
+<style>
+
+</style>
+
+<div class="container mx-auto p-3 sm:py-4">
+  <Header />
```

Comment:
I would destruct header component to smaller peaces. 
What the point to keep it together?
Header title it's just a text, random quote could be separate component and date is already dedicated